ncm-aiiserver/src/test/perl/options.t

```diff
@@ -0,0 +1,44 @@
+use strict;
+use warnings;
+use Test::More;
+use Test::Quattor qw(aii-options);
+use NCM::Component::aiiserver;
+use CAF::Object;
+use Readonly;
+
+
+=pod
+
+=head1 SYNOPSIS
+
+Basic test for C<ncm-aiiserver>
+
+=cut
+
+Readonly my @CONFIG_PARTS => qw(shellfe dhcp);
+Readonly my %CONFIG_ROOTS => (shellfe => '/software/components/aiiserver/aii-shellfe',
+                             dhcp => '/software/components/aiiserver/aii-dhcp');
+Readonly my %CONFIG_FILES => (shellfe => '/etc/aii/aii-shellfe.conf',
+                              dhcp => '/etc/aii/aii-dhcp.conf');
+
+$CAF::Object::NoAction = 1;
+
+my $comp = NCM::Component::aiiserver->new('ncm-aiiserver');
+my $cfg = get_config_for_profile('aii-options');
+$comp->Configure($cfg);
+
+for my $part (@CONFIG_PARTS) {
+    my $part_config = $cfg->getElement($CONFIG_ROOTS{$part})->getTree();
+    my $config_file = $CONFIG_FILES{$part};
+
+    # Be sure to do the same has the components in term of ordering
+    my $expected_contents = '^#.*\R#.*\R';
```

Comment:
Because of the header... I think it makes things unnecessary rigid to try to match the exact text. The regexp reflects what is valid (spaces around `=` are not).